packages/oca/src/types/TypeEnums.ts

```diff
@@ -0,0 +1,20 @@
+export enum BaseType {
```

Comment:
Not sure how relevant these really are anymore, if we have the option to remove them we should

Reply:
Ok, not to be pedantic but lets maybe name it to `CaptureBaseType` just so it's clear it's a type in that specific object

Reply:
Right, gonna do that.